_docs_integrate/migration-from-v4-to-v5.md

```diff
@@ -72,46 +73,58 @@ The aspects to be taken into account when migrating to version 5, which were bri
 
 ### DIDs as Addresses
 
-The address format changed from `<3-character realm><32- or 33-character base58-string>` to `did:e:<backbone-hostname>:dids:<22-character lowercase hex string>`.
+Each [Identity]({% link _docs_integrate/data-model-overview.md %}#identity) within enmeshed has a unique `address` for identification. The format of this `address` has changed from `<3-character realm><32- or 33-character Base58-encoded string>` to `did:e:<hostname of Backbone>:dids:<22-character lowercase hexadecimal string>`. Accordingly, the `realm` property of the Identity was removed as well. Furthermore, the two error codes `error.runtime.MultiAccount.WrongRealm` and `error.transport.identity.realmLength` have been removed.
 
 ### Removal of RelationshipChanges
 
-RelationshipChanges were removed. The only type of RelationshipChange used before was the CreationChange, its `content` has been moved to a new property of the Relationship - the `creationContent`. If this content has been created by responding to a [Request]({% link _docs_integrate/data-model-overview.md %}#request) of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate), the [content]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) changes as follows: The `<relationshipChange>.request.content` value was of type RelationshipCreationChangeRequestContent, the `<relationship>.creationContent` is now of type [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent).
-
-The Connector route for creating the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) is unchanged, the Connector routes using RelationshipChanges (accept RelationshipChange and reject RelationshipChange) were replaced with corresponding routes on the Relationship. No `content` must be sent with any of those operations. The operations are recorded in the new `auditLog` property of the Relationship.
+Previously, the history of changes made to a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) was stored in its `changes` property.
+The type of this property was given by an array of RelationshipChanges.
+Due to the inconvenience of providing new features for the management of Relationships using RelationshipChanges, the RelationshipChanges have now been removed.
+Consequently, the `changes` property of the Relationship was also removed.
```

Comment:
to make paragraphs you need a complete newline between two lines. This will be a big paragraph now. Maybe you want to insert more newlines here to make it more readable.

Reply:
Exactly, I did not intend to divide this paragraph into two or more paragraphs. The new lines are only used here because it was noted that it would be easier to review the text, as each sentence would then have its own line in the code.

However, I can try to split the text into two paragraphs if you prefer.

Reply:
> it would be easier to review the text, as each sentence would then have its own line in the code

for me this makes it way harder to know how it will look like rendered. But if you decided for this I shut up :D

Reply:
Hahaha I'm not a fan of using the line breaks to make the review easier either. But I can understand the argument, and it's probably a matter of getting used to it so it doesn't bother you anymore when imagining the rendering :D